src/wallet/walletdb.cpp

```diff
@@ -782,6 +791,25 @@ static DataStream PrefixStream(const Args&... args)
 static DBErrors LoadDescriptorWalletRecords(CWallet* pwallet, DatabaseBatch& batch, int last_client) EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet)
 {
     AssertLockHeld(pwallet->cs_wallet);
+    DBErrors result = DBErrors::LOAD_OK;
+
+    // Get Active HD Key
+    LoadResult active_hdkey_res = LoadRecords(pwallet, batch, DBKeys::ACTIVEHDKEY,
+        [] (CWallet* pwallet, DataStream& key, DataStream& value, std::string& err) EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet) {
```

Comment:
In 8bb7a8b06:

`EXCLUSIVE_LOCKS_REQUIRED` annotation is not required.

Reply:
Done